src/main/java/edu/cornell/kfs/coa/document/AccountReversionMaintainableImpl.java

```diff
@@ -257,4 +263,15 @@ protected boolean shouldIncludeField(Field field) {
         return includeField;
     }
     
+    @Override
+    public void doRouteStatusChange(DocumentHeader documentHeader) {
```

Comment:
I noticed that the various `doRouteStatusChange()` overrides are set up to clear the cache on every status change. During your Pre-KEW testing, you may want to verify whether the cache clearing should be limited to certain specific status changes (ENROUTE, PROCESSED, DISAPPROVED, etc.).